src/speak/options.rs

```diff
@@ -6,22 +6,20 @@
 
 use serde::{ser::SerializeSeq, Serialize};
 
-/// Used as a parameter for [`Speak::speak_to_file`](crate::speak::text_to_speech::Speak::speak_to_file) and similar functions.
+/// Used as a parameter for [`Speak::speak_to_file`](crate::Speak::speak_to_file) and similar functions.
 #[derive(Debug, PartialEq, Clone)]
 pub struct Options {
     model: Option<String>,
     encoding: Option<String>,
     sample_rate: Option<u32>,
     container: Option<String>,
```

Comment:
Before we release 0.6.0, can we introduce (non-exhaustive) enums (with a fallback `String` variant) for model, encoding, and container?